ee/modules/160-multitenancy-manager/templates/user-resources/default-project-template.yaml

```diff
@@ -39,6 +45,9 @@ spec:
                   pattern: '^[0-9]+(\.[0-9]+)?(E|P|T|G|M|k|Ei|Pi|Ti|Gi|Mi|Ki)?$'
             limits:
               type: object
+              required:
```

Comment:
limits should not be required. We have to fix the templates here, to handle empty limits

Reply:
these options are required because of [this](https://github.com/deckhouse/deckhouse/blob/19d74a94b5ee079ff639295a8e85252c601d1a65/ee/modules/160-multitenancy-manager/templates/user-resources/secure-with-dedicated-nodes-project-template.yaml#L234)